differ/diff.go

```diff
@@ -56,11 +57,17 @@ func (d *Differ) run(command string, args ...string) error {
 		return fmt.Errorf("buildifier: %s: %v", command, err)
 	}
 
-	// Assume bash reported anything else worth reporting.
-	// As long as the program started (above), we don't care about the
-	// exact exit status. In the most common case, the diff command
-	// will exit 1, because there are diffs, causing bash to exit 1.
-	return cmd.Wait()
+	// Assume that the diff program adhere to ordinary diff commands
+	// exit codes i.e. 0 no differences, 1 differences found, >1 an
+	// error occoured.
+	fmt.Fprintf(os.Stdout, "differ: %s\n", cmd)
+	if err := cmd.Wait(); err != nil {
+		if status, ok := err.(*exec.ExitError).Sys().(syscall.WaitStatus); ok && status.ExitStatus() != 1 {
```

Comment:
This will crash is the `err` is not of type `ExitError`.  According to the documentation that may happen: https://golang.org/pkg/os/exec/#Cmd.Wait

Also I can't find the documentation for the `.Sys()` method, can it return nill or an object of something else than type `syscal.WaitStatus`?